Confus/Confus/Player.h

```diff
@@ -60,8 +64,19 @@ namespace Confus
         /// <summary> Whether the player is currently walking or not </summary>
         bool m_Walking = false;
 
+        bool m_IsMainPlayer = false;
```

Comment:
I think there's already a bool for this, m_MainPlayer, or is this for something different?